gradle/libs.versions.toml

```diff
@@ -28,5 +28,5 @@ vavr = "io.vavr:vavr:0.10.4"
 # where they aren't really `apply`-ed as plugins, but rather added to classpath as plugin packages;
 # see https://docs.gradle.org/current/samples/sample_convention_plugins.html#things_to_note.
 [plugins]
-jetbrains-grammarkit = "org.jetbrains.grammarkit:2022.3.2.1"
+jetbrains-grammarkit = "org.jetbrains.grammarkit:2022.3.2.1-1-SNAPSHOT"
```

Comment:
To be replaced with workaround from https://github.com/JetBrains/gradle-grammar-kit-plugin/issues/168#issuecomment-1914780767

Reply:
(the current one won't work locally unless someone publishes `gradle-grammar-kit-plugin`, as in `.circleci/config.yml` above)